src/resources.js

```diff
@@ -396,13 +409,15 @@ export class Resources {
    * `overflowCallback` method on the target resource with the height value.
    * Overflow callback is expected to provide the reader with the user action
    * to update the height manually.
+   * If the height is successfully updated then the opt_callback is called.
    * @param {!Element} element
    * @param {number} newHeight
+   * @param {function=} opt_callback A callback function.
    * @protected
    */
-  requestChangeHeight(element, newHeight) {
+  requestChangeHeight(element, newHeight, opt_callback) {
```

Comment:
Can we rename this to changeHeightIfAllowed and update the docs. Confusingly `changeHeight `'s docs start with "Requests…"